src/tools/driver/logcat.dart

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2015, the Fletch project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE.md file.
+
+library driver.logcat;
+
+import 'dart:io';
+import 'dart:async';
+import 'dart:convert';
+import 'help_text.dart' as help;
+import 'cmd_line_utils.dart' as cmd;
+
+main(List<String> args) {
+  LogcatServer.performFromArgs(args);
+}
+
+class LogcatServer {
+  static performFromArgs(List<String> args) {
+    if (args.length != 3) {
+      print("logcat requires exactly 3 arguments");
+      print(help.LOGCAT_HELP_TEXT);
+      return;
```

Comment:
Do you want to exit with a non-zero exit code? Maybe use exit(1) rather than return.